samples/bluetooth/hci_usb_h4/prj.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_GPIO=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_HCI_RAW_RESERVE=1
+
+CONFIG_USB=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_LOG=y
+CONFIG_USB_DRIVER_LOG_LEVEL_ERR=y
+CONFIG_USB_DEVICE_BT_H4=y
```

Comment:
What do you think about to merge it with samples/bluetooth/hci_usb and add configuration overlays for USB_DEVICE_BLUETOOTH and USB_DEVICE_BT_H4?

Reply:
I think for USB_DEVICE_BT_H4 alone it was simpler, but I don't oppose to transform it on a overlay it just seems confusing since that would create several modes of operation to it.

Reply:
The hci_usb_h4 is not using the standard descriptors, it just has the bulk endpoints and which is why it uses a dedicated PID.